src/VoteContainer/VoteContainer.js

```diff
@@ -0,0 +1,11 @@
+import './VoteContainer.css';
+
+function VoteContainer({ votes }) {
+  return (
+    <div className="VoteContainer">
+      <p>{votes}</p>
+    </div>
+  );
+}
+
+export default VoteContainer;
```

Comment:
Thank you for adding the vote container, this will help when we add the vote buttons.